pkg/kubelet/nodestatus/setters.go

```diff
@@ -83,13 +83,13 @@ func NodeAddress(nodeIPs []net.IP, // typically Kubelet.nodeIPs
 			if err := validateNodeIPFunc(nodeIP); err != nil {
```

Comment:
something bad happened in this file and it reverted your structured logging changes :frowning_face: 

I think in your rebase you picked up a later commit and it overwrote.

going to throw a hold on this so you can sort it out. :heart: 

/hold